scripts/deploy-to-app-center.sh

```diff
@@ -117,6 +118,7 @@ appcenter distribute release \
 --file "$TMP_DIR"/MiniApp_Example.ipa \
 --quiet
```

Comment:
if you want to log more info you can also remove the `--quiet`

Reply:
Nah. That would be too noisy. We can remove this whenever we need it.